packages/project-utils/package.json

```diff
@@ -41,6 +42,7 @@
     "get-yarn-workspaces": "1.0.2",
     "glob": "^7.1.2",
     "html-webpack-plugin": "4.0.0-beta.5",
+    "https": "^1.0.0",
```

Comment:
Hm... why `https` here?

Reply:
BTW as mentioned in other comment, native Node modules do not need to be declared as dependencies. Feel free to remove this.

Reply:
same as https://github.com/webiny/webiny-js/pull/2128#discussion_r776326749